src/item/class/sheet.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+import { FabulaUltimaItemSheet } from "../sheet.js";
+
+export class FabulaUltimaClassSheet extends FabulaUltimaItemSheet {
+  static get defaultOptions() {
+    return mergeObject(super.defaultOptions, {
+      classes: ["fabula-ultima", "sheet", "item"],
+      width: 660,
+      height: 740,
+      resizable: false,
+    });
+  }
+
+  get template() {
+    return "systems/fabula-ultima/templates/item/class/sheet.hbs";
+  }
+  async getData() {
```

Comment:
NB to match the spacing between the other two methods:
```suggestion
  }
  
  async getData() {
```